stock_picking_package_preparation/model/stock_picking_package_preparation.py

```diff
@@ -161,7 +161,7 @@ def action_done(self):
             raise exceptions.Warning(
                 _('The package has not been generated.')
             )
-        self.picking_ids.do_transfer()
+        self.picking_ids.filtered(lambda x: x.state != 'done').do_transfer()
```

Comment:
Theoretically yes, but I didn't found a method to put in pack a cancelled picking
I add the filter anyway

Reply:
@sergiocorato I think the @gurneyalex remark makes sense. I remember a fix I did to avoid multiple picking validation (If different users click on button).

And Odoo did the same in https://github.com/odoo/odoo/blob/9e8f70e4849b0eeaca8b5cf51372ecfa23dc561b/addons/stock/stock.py#L1457

Reply:
done!